python/lsst/ts/linearstage/controllers/igus_dryve.py

```diff
@@ -301,16 +314,16 @@ async def set_drive_settings(self):
         # sendCommand(bytearray([0, 0, 0, 0, 0, 15, 0, 43, 13, 1, 0, 0, 96,
         #                        146, 1, 0, 0, 0, 2, 112, 23]))
 
-        _feed_constant = round(self.config.feed_rate * _multi_factor)
+        _feed_constant: int = round(self.config.feed_rate * _multi_factor)
         if _feed_constant > 2**16:
             # Catch this here because diagnosing the error is tough
             raise NotImplementedError(
                 f"Value for given for feed constant of {self.config.feed_rate}"
                 f" exceeds limit of {(2 ** 16) / _multi_factor}"
             )
-        byte19 = _feed_constant & 0b11111111
-        byte20 = _feed_constant >> 8
-        telegram = (
+        byte19: int = _feed_constant & 0b11111111
+        byte20: int = _feed_constant >> 8
+        telegram: tuple = (
```

Comment:
These three lines are excellent examples of magic numbers that should be defined in constants.